### ISSUE TITLE
eComm onboarding - store profiler - business details: hide free extensions

### ISSUE THREAD

**Issue by pmcpinto** [Automattic/wc-calypso-bridge#607]
We shouldn't display the free extensions in the business details step: https://d.pr/i/lAgh3C

It seems to be a regression as this was already fixed in the past

**Comment by timmyc** [comment]
@pmcpinto I'm thinking what might have happened here for you is you got redirected out of calypsoify, and as such the `calhypsoify.css` we load specific for the ecommerce plan. I could verify this happening locally by visiting`/wp-admin` to clear out the calypsoify cookie and user meta, then visiting the profiler directly via `/wp-admin/admin.php?page=wc-admin&reset_profiler=1`

Oddly enough, just now when I clicked on the Store link from Calypso for a test store, I was properly redirected to calysoify vs what I encountered when we were chatting about this issue earlier today.

We could explore **forcing** our fixes to always be applied to the onboarding flow for e-commerce plan customers, but that would be a bit more work. But I think long-term that is likely going to have to happen with the navigation project.

As such I'm leaning towards letting this exist as a known issue - and TBH this isn't a regression, just how calypsoify works. If you "exit out" of calypsoify, our logic in calypso bridge on purpose doesn't load in the related assets which enable the custom view.
